src/player.js

```diff
@@ -130,9 +130,8 @@ class Player {
         }
 
         if (screenfull.isEnabled) {
-            const exitFullscreen = () => screenfull.exit();
-
-            screenfull.on('fullscreenchange', () => {
+            this.fullscreenchangeHandler = () => {
+                const exitFullscreen = () => screenfull.exit();
```

Comment:
`const exitFullscreen = () => screenfull.exit();` should be left out of the event listener,
otherwise you're essentially creating new `exitFullscreen` functions on `fullscreenchange` events making the `removeCallback(this, 'event:exitFullscreen', exitFullscreen);` not work as it expects the specific function that was added with `storeCallback(this, 'event:exitFullscreen', exitFullscreen);`